frontend/src/component/Account/PersonalData.jsx

```diff
@@ -136,6 +146,9 @@ export class PersonalData extends React.Component {
 }
 
 PersonalData.propTypes = {
+  userEmail: PropTypes.string,
+  userUsername: PropTypes.string,
+  userPublicName: PropTypes.string,
```

Comment:
```suggestion
  userEmail: PropTypes.string.isRequired,
  userUsername: PropTypes.string.isRequired,
  userPublicName: PropTypes.string.isRequired,
```

Reply:
we don't need the .isRequired in this case